### PR TITLE
yt-dlp: new package at 2021.10.22

### PR DESCRIPTION
**Pull request by ioraff**
## Description of package
A youtube-dl fork with additional features and fixes

## Installed manifest of package

```
/var/db/kiss/installed/yt-dlp/version
/var/db/kiss/installed/yt-dlp/sources
/var/db/kiss/installed/yt-dlp/manifest
/var/db/kiss/installed/yt-dlp/files/youtube-dl
/var/db/kiss/installed/yt-dlp/files/
/var/db/kiss/installed/yt-dlp/depends
/var/db/kiss/installed/yt-dlp/checksums
/var/db/kiss/installed/yt-dlp/build
/var/db/kiss/installed/yt-dlp/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/bin/yt-dlp
/usr/bin/youtube-dl
/usr/bin/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify the @kiss-community maintainers if I can no longer maintain this package.

### CODE REVIEW

File: community/yt-dlp/files/youtube-dl [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec yt-dlp --compat-options "$@"
```
Comment (Riteo) [inline community/yt-dlp/files/youtube-dl:2]: I think you forgot to specify `youtube-dl` to `--compat-options`.

File: community/yt-dlp/depends [hunk 2]
```diff
@@ -0,0 +1 @@
+python
```
Comment (Riteo) [inline community/yt-dlp/depends:1]: This package also requires `zip`, I can't build without it. It doesn't seem to need it on runtime though so you can make it a make dependency.

**Review by Riteo — COMMENTED** [review]
There are some issues with this package, see the comments I made.

**Comment by ioraff** [comment]
Fixed in #797. Thanks.

**Pull request merged**